textproc/cfonts/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo  1.0
+
+github.setup        DominikWilkowski cfonts 1.0.2rust v
```

Comment:
`github.setup`'s format looks like `<author> <project_name> <version> <version_prefix> <version_suffix>`

So you can break the `1.0.2rust` version in this way:
```
github.setup        DominikWilkowski cfonts 1.0.2 v rust
```

Reply:
fixed